frontend/src/assets/css/animations.scss

```diff
@@ -0,0 +1,11 @@
+$anim-timing-fn-ui: ease-in-out;
```

Comment:
this animation variables will be used somewhere else throughout the project or only in one place? do we need to place them in src/assets?

Reply:
If you asking me, i'd answer yes, this will be used by anyone, who would make ui animations.